src/app/[locale]/(pages)/accedi/page.tsx

```diff
@@ -114,7 +123,9 @@ const Access = (): React.ReactElement => {
                       width: '100%',
                       height: '50px',
                     }}
-                    onClick={() => setOpenDialog(true)}
+                    onClick={() => {
+                      setOpenDialog(true);
+                    }}
```

Comment:
Why do we have this change? 🤔. Don't we have in our lint config the "single statement arrow function" as the preferred way?

Reply:
In not, maybe we should consider to add it, so that to be aligned with the lint rules in the IO app repo.

Reply:
During testing, I had placed the trackEvent inside the button event and had used curly braces to handle multiple functions, then I forgot them, sorry